blob/gcsblob/example_test.go

```diff
@@ -22,27 +22,11 @@ import (
 	"gocloud.dev/blob"
 	"gocloud.dev/blob/gcsblob"
 	"gocloud.dev/gcp"
-	"golang.org/x/oauth2/google"
 )
 
-// jsonCreds is a fake GCP JSON credentials file.
-const jsonCreds = `
-{
-  "type": "service_account",
-  "project_id": "my-project-id"
-}
-`
-
 func Example() {
 	ctx := context.Background()
-
-	// Get GCP credentials.
```

Comment:
Can we leave the comment?

Reply:
Done.

Reply:
Not just this line, the whole thing. It explains that we're using fake credentials here, but that they should use `DefaultCredentials` or one of the other alternatives from the link.

Reply:
Part of why I wanted this change was to get rid of the copy/pasted comments and have that text in one place, right above the definition of `gcp.FakeDefaultCredentials`.